00-Overview/readme.md

```diff
@@ -1,21 +1,21 @@
 Overview
 ========
 
-> Git is distributed revision control system with an emphasis on speed, data integrity, and support for distributed, non-linear workflows.
+> Git is a distributed revision control system with emphasis on speed, data integrity, and support for distributed, non-linear workflows.
 
-Now, this is great and all, but what it means? Here are some points some people have problem getting about git:
+It is great and all, but what it means? Here are some points some people have problem getting about git:
 
-* everything related to git repository is stored in `.git` folder in root of your project; you can just copy project folder and git repository will travel with it
+* everything related to the git repository is stored in the `.git` folder in the root of your project; you can simply copy the project folder and the git repository will travel with it
 * from technical standpoint, there is no central repository, all repositories are equal
-* usually there is one `upstream`, but which repository it is depends on agreement between project members
+* there usually is one `upstream`, but which repository it is depends on agreement between project members
 * this has some (maybe) unexpected side effects
 	* you cannot restrict read access to specific files, folders, branches; if you want someone to clone from you, he has to be able to get everything
 	* everyone has complete history of the project
 
 All repositories are equal
 --------------------------
 
-* while you do clone from somewhere, you are not restricted to pushing/pulling from there
+* while you do clone from somewhere, you are not restricted to pushing to/pulling from there
```

Comment:
On , KyleFord76063 wrote:
>
> [..]
> 
> Not sure the "to" is needed.  Could rephrase like this:
> "...not restricted to "pushing / pulling" from there"

I like the Kyle's version more :)

-- 
There are only two hard things in Computer Science:
cache invalidation, naming things and off-by-one errors.